pymc/tests/test_distributions.py

```diff
@@ -2616,6 +2616,16 @@ def test_interpolated_transform(self, transform):
             assert not np.isfinite(m.compile_logp()({"x": -1.0}))
             assert not np.isfinite(m.compile_logp()({"x": 11.0}))
 
+    @pytest.mark.skip("Unsure how to use this test fixture")
+    def test_grw(self):
+        self.check_logp(
+            pm.GaussianRandomWalk,
+            R,
+            {"mu": R, "sigma": Rplus, "steps": Nat},
+            lambda value, mu, sigma: sp.norm.logpdf(value, mu, sigma).cumsum().sum(),
```

Comment:
This has to be updated to include `init`

Reply:
I thought init was not included in the logp calc though and is accounted for elsewhere?

Reply:
It is included again since the last couple of commits.

Reply:
How can the logp for init be included here? I'm confused because the init distribution is declared elsewhere and passed to GRW, but the logp for the init dist is calculated within the grws logup. 

That all makes sense up until this fixture where its not clear how to properly do that in the text context. Any ideas?